doc/nrf/libraries/nfc/rpc/index.rst

```diff
@@ -0,0 +1,90 @@
+.. _nfc_rpc:
+
+NFC Remote Procedure Call
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The :ref:`NFC <ug_nfc>` Remote Procedure Call (RPC) solution is a set of libraries that allows using the NFC stack running entirely on a separate device or CPU.
+
+Overview
+********
+
+The solution allows calling the NFC API (both :ref:`Type 2 Tag <lib_nfc_t2t>` and :ref:`Type 4 Tag <lib_nfc_t4t>`) on a different CPU or device.
+This is accomplished by running the full NFC functionality on one device and serializing the API from another device.
+Use this solution when you do not want your firmware to include the NFC stack, for example to offload the application CPU, save memory, or to be able to build your application in a different environment.
+
+Implementation
+==============
+
+The NFC RPC solution consists of the following components:
+
+  * NFC RPC Client and common software libraries.
+    These libraries serialize the NFC API and enable RPC communication, and need to be part of the user application.
+  * NRF RPC Server, the Type 4 or/and Type 2 Tag library and common libraries that enable communication with NFC RPC Client.
+    This software needs to run on a device or CPU that has NFC radio hardware peripheral.
+  * Build system files that automate building NFC RPC.
+
+You can add support for serializing NFC-related custom APIs by implementing your own client and server procedures.
+You can use the following files as examples:
+
+  * :file:`subsys/nfc/rpc/client/nfc_rpc_t2t_client.c`
+  * :file:`subsys/nfc/rpc/server/nfc_rpc_t2t_server.c`
+
+Requirements
```

Comment:
Don't we want to mention how to actually enable NFC RPC (CONFIG_NFC_RPC_CLIENT) in a project?